Formula/mdoc.rb

```diff
@@ -0,0 +1,41 @@
+class Mdoc < Formula
+  desc "Command-line tool for writing scientific documents"
+  homepage "https://kmaasrud.com/mdoc"
+  url "https://github.com/kmaasrud/mdoc/archive/0.3.1.tar.gz"
+  sha256 "6807701201e0149bdb09d2ca7f32590a646dfb238823a5f448384a844e8b78f1"
+  license "MIT"
+  head "https://github.com/kmaasrud/mdoc.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
There don't seem to be any pre-releases so tag checking seems good enough.

Reply:
Do you want me to use the regex method like this?

```ruby
livecheck do
  url :stable
  regex(/^v?(\d+(?:\.\d+)+)\.t/i)
end
```

Reply:
Don't specify anything and use the default